config/development.js

```diff
@@ -50,6 +50,19 @@ module.exports = {
       "-----END RSA PRIVATE KEY-----",
   },
 
+  botToken: {
+    botPublicKey:
+      "-----BEGIN PUBLIC KEY-----\n" +
```

Comment:
Do you know if we need this in development?

Because without a private key, this public key is useless, 

Our public key will be saved in the discord bot, in development, we can update the readme and ask the developers to get their own public and private key pair and test it.

Reply:
Yeah make sense but it would be easier if we direct them to use the privateKey from tests to generate the jwt token and developers can easily pass that jwt token to headers to test the API.

Developers can make mistakes as adding the key requires formatting and this will make them reach out to people to fix the bug. If we provide them formatted keys, they only need copy paste it and make the things work.